src/main/java/de/tum/in/www1/artemis/service/feature/FeatureToggleService.java

```diff
@@ -49,6 +54,10 @@ public void enableFeature(Feature feature) {
      * @param feature The feature that should be disabled
      */
     public void disableFeature(Feature feature) {
+        if (feature == Feature.ExamLiveStatistics) {
```

Comment:
This looks good for now, but if multiple features get such special cases these methods can get quite long.

You could also add Method to the feature-enum directly, making the ExamLiveStaticsitcs-enum override the method. And here you just call feature.onDisable();

Reply:
I am not sure if this works well with external services :thinking:

Reply:
oh that's a good point. So I guess we have to keep it like this.

Reply:
Yes i think thats the best solution at the moment